content/700-optimize/400-recommendations/700-avoid-varchar.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Using @db.VarChar(n)'
+metaTitle: 'Optimize Recommendations: Avoid usage of `@db.VarChar(x)`'
```

Comment:
> Avoid usage of `@db.VarChar(x)`
Should we change this to be consistent with the rest have have it be: 
Avoid usage of `@db.VarChar(n)`
?

Reply:
```suggestion
metaTitle: 'Optimize Recommendations: Avoid usage of `@db.VarChar(n)`'
```